src/libraries/Microsoft.PowerFx.Core/Public/Values/ValidFormulaValue.cs

```diff
@@ -17,5 +18,11 @@ internal ValidFormulaValue(IRContext irContext)
             : base(irContext)
         {
         }
+
+        // for custom types
+        protected ValidFormulaValue(FormulaType type)
```

Comment:
What's the goal of this? Can you provide some examples of why you'd need this added?

Reply:
One simple example would be `PictureValue`, but you can imagine any type that only custom functions or consumers would be able to process. I'm guessing it would be paired with `ExternalType`.

Otherwise, how would one feed PowerFX values that don't match any of the built-in types?